Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import "./Footer.css";
+
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous"></link>
+
+
+const commentClicked = () => {
+    alert('You clicked me!')
+}
+
+const reTweet = () => {
+    alert('You Re-tweeted this!')
+}
+
+const heart = () => {
+    alert('You liked this!')
+}
+
+
+const Footer = () => {
+    return (        <div className='icons'> 
+
+            <i class="far fa-comment" onClick = {commentClicked} ></i>
```

Comment:
good job getting font awesome up and going. Play around a bit with the styles and see what else you can do to it. Love the onClick handler as well. I'm impressed for sure.